src/addNewTask.js

```diff
@@ -0,0 +1,15 @@
+const form = document.querySelector('form');
+const input = document.querySelector('.placeholder');
+
+const addNewTask = (arr, item) => {
+  function Task(item) {
+    this.item = item;
+    this.completed = false;
+    this.index = arr.length;
+  }
+  const task = new Task(item);
+  arr.push(task);
+  localStorage.setItem('task', JSON.stringify(arr));
+};
```

Comment:
- [OPTIONAL] When adding a task, the `index` of the first task should be set to number `1`, the `index` of the second task should be set to number `2` and so on. You have set the `index` to be equal to the **_length of the array_**. This means that the `index` of the first task would have a value of `0` because the array has no item in it as of when the first task is being created. To fix this, you need to set the logic for the index to be 💻:

`this.index = arr.length + 1;` 

I would advise you to implement it this way because it is a requirement for this milestone:

![image](https://user-images.githubusercontent.com/77106962/236256880-ee8770e3-893c-4a70-b141-b34e04c7c2a4.png)

Reply:
Issue resolved. 
Thank you for the suggestions.